runtime/runtime-params-estimator/src/gas_cost.rs

```diff
@@ -394,7 +406,14 @@ impl fmt::Debug for GasCost {
                     self.io_w_bytes.to_f64().unwrap()
                 )
             }
-            GasMetric::Time => fmt::Debug::fmt(&Duration::from_nanos(self.time_ns.to_integer()), f),
+            GasMetric::Time => {
+                if self.time_ns >= 1.into() {
+                    fmt::Debug::fmt(&Duration::from_nanos(self.time_ns.round().to_integer()), f)
+                } else {
+                    // Sometimes, dividing costs yields results smaller than one ns.
+                    write!(f, "{}ps", (self.time_ns * 1000).to_integer())
```

Comment:
What would be an example of such cost? And are we talking about `0.5ns` here, or about `0.05ns`? Costs *significantly* smaller than ns-per-byte would seem suspicious to me.